### PR TITLE
Make operations plugin work with external fragments

### PR DESCRIPTION
**Pull request by ertrzyiks**
addresses #30

 - [x] Reference the related issues, if any.
 - [x] Update the changelog by adding entries to the "Unreleased" section.
 - [x] Add tests, if that makes sense. 
 - [x]  Add yourself to the list of contributors in the readme.

### CODE REVIEW

File: packages/graphql-codegen-factories/src/operations/__tests__/plugin.ts [hunk 1]
```diff
@@ -117,6 +117,54 @@ describe("plugin", () => {
...
+        me: User!
+      }
+    `);
+
+    const externalFragment = parse(/* GraphQL */ `
+      fragment UserFragment on User {
+        id
+        username
+      }
+    `).definitions[0] as unknown as FragmentDefinitionNode;
```
Comment (ertrzyiks) [inline packages/graphql-codegen-factories/src/operations/__tests__/plugin.ts:137]: Is there better way to build a `FragmentDefinitionNode` ?

**Review by zhouzi — APPROVED** [review]
Super clean, thanks @ertrzyiks 🙏 I just have a few minor suggestions.

File: changelog.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## Fixed
+- Fix support for external fragments
```
[the remark this answers is not in the document]
Comment (ertrzyiks) [reply changelog.md:7]: Fixed

File: packages/graphql-codegen-factories/src/operations/__tests__/plugin.ts [hunk 3]
```diff
@@ -117,6 +117,55 @@ describe("plugin", () => {
...
+    `);
+
+    const fragments = parse(/* GraphQL */ `
+      fragment UserFragment on User {
+        id
+        username
+      }
+    `);
+
+    const allFragment = fragments.definitions.filter(
```
[the remark this answers is not in the document]
Comment (ertrzyiks) [reply packages/graphql-codegen-factories/src/operations/__tests__/plugin.ts]: Done

File: packages/graphql-codegen-factories/src/operations/plugin.ts [hunk 4]
```diff
@@ -18,14 +18,15 @@ export const plugin: PluginFunction<
...
+  const allFragments: LoadedFragment[] = [
+    ...fragmentDefinitions.map((fragmentDefinition) => ({
       node: fragmentDefinition,
       name: fragmentDefinition.name.value,
       onType: fragmentDefinition.typeCondition.name.value,
       isExternal: false,
-    })
-  );
+    })),
+    ...(config.externalFragments || []),
```
[the remark this answers is not in the document]
Comment (ertrzyiks) [reply packages/graphql-codegen-factories/src/operations/plugin.ts:30]: Done

**Pull request merged**